eventbridge-pipes-amazonmq-to-eventbridge-sam/eventbridge-pipes-amazonmq-to-eventbridge-sam-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Can you rename this file to example-pattern.json please?

Reply:
Renamed to `example-pattern.json` in commit 9175bc0dbb7bf99a2290fc4e9246968277135cc1